app/controllers/app.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { findTopics } = require('../models/app.models');
+
+const getTopics = (request, response, next) => {
+  findTopics()
+    .then((result) => {
+      response.status(200).send(result.rows);
```

Comment:
slight issue here, the response array must be sent on a property key

i.e.  `.send({ topics: result.rows});` 

This is convention for sending responses so that the data is stored on semantic key names for the end users to access - when you change this make sure to also update the test to deconstruct out this object key